pex/variables.py

```diff
@@ -275,6 +275,19 @@ def PEX_PYTHON_PATH(self):
     """
     return self._get_string('PEX_PYTHON_PATH', default=None)
 
+  @property
+  def PEX_EXTRA_SYS_PATH(self):
+    """String
+
+    A colon-separated string containing paths to add to the runtime sys.path.
```

Comment:
It would be good to add docs both here and to `PEX_INHERIT_PATH` that explain how these two options interact.

As far as I can tell, the interaction is fine? "inherit" talks about what to do with the existing path defined by the interpreter, which this allows items to be added after that logic has run.

You might also want to describe when you should use this vs `PEX_PATH`, which also allows the PYTHONPATH to be expanded at runtime (but with pexes rather than loose files). Related: should we be using PEX_PATH in some way here rather than loose files?

Reply:
In the pytest runner use case, we don't want to build a sources pex, for performance reasons.